### PR TITLE
fix: #953 paginator default page

### PR DESCRIPTION
**Pull request by J0taFerreira**
- Changed default paginator to 50 pages
-

### CODE REVIEW

**Review by ianliuwk1019 — APPROVED** [review]
There are some Sonar issues, could you take a look? Otherwise is good.
Thanks Jota!

File: frontend/src/components/managePermissions/UserDataTable.vue [hunk 1]
```diff
@@ -79,7 +80,7 @@ function deleteAssignment(assignment: FamApplicationUserRoleAssignmentGet) {
         acceptLabel: 'Remove',
         acceptClass: 'p-button-danger',
         accept: () => {
-            emit('deleteUserRoleAssignment', assignment);
+            emits('deleteUserRoleAssignment', assignment);
```
Comment (ianliuwk1019) [inline frontend/src/components/managePermissions/UserDataTable.vue:83]: Maybe lets just name it "emit" (no 's'), just to be consistent:
image

Comment (J0taFerreira) [reply frontend/src/components/managePermissions/UserDataTable.vue]: just did. Thanks, Ian!

File: frontend/src/components/managePermissions/UserDataTable.vue [hunk 2]
```diff
@@ -16,6 +16,11 @@ import Button from '@/components/common/Button.vue';
 
 import type { FamApplicationUserRoleAssignmentGet } from 'fam-api/dist/model/fam-application-user-role-assignment-get';
 
+type emit = (
+        e: 'deleteUserRoleAssignment',
+        item: FamApplicationUserRoleAssignmentGet
+) => void
+
```
[the remark this answers is not in the document]
Comment (J0taFerreira) [reply frontend/src/components/managePermissions/UserDataTable.vue:23]: I'll take another look, ain't this the same way that was implemented before?

[the remark this answers is not in the document]
Comment (J0taFerreira) [reply frontend/src/components/managePermissions/UserDataTable.vue:23]: sonar cloud was complaining that: "Type literal has only a call signature, you should use a function type instead". 

function_type

[the remark this answers is not in the document]
Comment (ianliuwk1019) [reply frontend/src/components/managePermissions/UserDataTable.vue:23]: Hmm.... they are essentially the same from what I see, but I fee sonar seems to be too strict.

**Review by ianliuwk1019 — APPROVED** [review]

**Pull request merged**